test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_notification.feature

```diff
@@ -225,29 +268,56 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
       | http://www.foo.bar./       |
       | http://.www.foo.bar./      |
```

Comment:
the first case is added, but the second, port is not an integer is covered in the line `http://localhost:dsfsdf`